worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/ClipboardTransformFuser.java

```diff
@@ -41,7 +40,7 @@
  * @see Clipboard
  * @see Transform
  */
-class FlattenedClipboardTransform {
+public class ClipboardTransformFuser {
```

Comment:
This class may need additional rethinking. It definitely wasn't meant to be exposed in the current state.
Aside from the name (bake pls), it would probably be more useful to API users (including WE internally) to just provide the resultant Clipboard object instead of just exposing copyTo. This would also prevent it from being misused as a pastebuilder as a lot of clipboard manipulation can (and should) be done off main thread, and pre-baking should not be done at paste-time.